src/ripple/ledger/impl/PaymentSandbox.cpp

```diff
@@ -172,7 +172,11 @@ PaymentSandbox::balanceHook (AccountID const& account,
         }
     }
 
-    assert (!isXRP(issuer) || adjustedAmt >= beast::zero);
+    if (isXRP(issuer) && adjustedAmt < beast::zero)
+    {
+        adjustedAmt.clear();
```

Comment:
I figured it was probably nothing to worry about.  I just thought it would be nice to leave a comment in the source code about why it's okay.